src/rest/routes/subscriptions/delete.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, Router } from "express";
+import { errorWrapper } from "../../helper";
+import { removeSubscription } from "utils/jira-utils";
+import { Installation } from "models/installation";
+
+
+export const SubscriptionDeleteRouter = Router({ mergeParams: true });
+
+
+/**
+ * This delete endpoint only handles Github cloud subscriptions
+ */
+SubscriptionDeleteRouter.delete("/", errorWrapper("SubscriptionDelete", async (req: Request, res: Response) => {
+	// TODO: Get the github app id for GHE subscriptions
+	const gitHubAppId = undefined;
+	const subscriptionId: number = Number(req.params.subscriptionId);
+	const { installation } = res.locals as { installation: Installation; };
+
+	// Passing 0 as the `ghInstallationId` cause its not gonna be used anyways
```

Comment:
`Passing 0 as the `ghInstallationId` cause its not gonna be used anyways` <-- if so, why not take `undefined` for now?

Or better, write three functions
```javascript
export function removeSubscriptionById(installation, subscriptionId) {
  return doRemoveSubscription(sub: Subscription);
}
export function removeSubscriptionGHInstallationIDAndAppId(installation, ghInstId, ghAppId) {
  return doRemoveSubscription(sub: Subscription);
}
//__do_not_export__this
function doRemoveSubscription(sub: Subscription) {
  //do real stuff
}
```

Reply:
Updated the type and passed `undefined`.